app/src/main/java/com/mk/tasky/agenda/home/presentation/HomeScreen.kt

```diff
@@ -27,18 +29,41 @@ import java.time.format.DateTimeFormatter
 @Composable
 fun HomeScreen(
     redirect: (HomeAgendaType, LocalDateTime) -> Unit,
+    options: (HomeItemOptions, String) -> Unit,
+    onLogout: () -> Unit,
     viewModel: HomeViewModel = hiltViewModel()
 ) {
     val state = viewModel.state
     val context = LocalContext.current
 
+    LaunchedEffect(true) {
+        viewModel.onEvent(HomeEvent.OnRefreshAgenda)
```

Comment:
Why not put this in `init` of your ViewModel?

Reply:
Explained over whatsapp (init doesnt get called due to backstack)